### PR TITLE
Rewrite clients with OpenAPI Generator

### PR DESCRIPTION
**Pull request by moririnson**
Rewrite codes with OpenAPI Generator.

ref: https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/php.md

### CODE REVIEW

File: .github/workflows/php-checks.yml [hunk 1]
```diff
@@ -9,12 +9,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.2, 7.3, 7.4, 8.0, 8.1]
+        php: [8.1, 8.2]
```
Comment (tokuhirom) [inline .github/workflows/php-checks.yml:12]: Note: PHP SDK team will drop the old version support.

File: README.md [hunk 2]
```diff
@@ -188,11 +188,6 @@ Please refer [CurlHTTPClient](/src/LINEBot/HTTPClient/CurlHTTPClient.php) that i
 See also
 --
 
-### [line-bot-sdk-tiny](./line-bot-sdk-tiny)
```
Comment (tokuhirom) [inline README.md:191]: Note: In 2023, PHP developers can use composer. There's no reason to provide a simple version of the SDK.

File: src/clients/channel-access-token/test/Model/IssueChannelAccessTokenResponseTest.php [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        $this->markTestIncomplete('Not implemented');
+    }
+
+    /**
+     * Test attribute "accessToken"
+     */
+    public function testPropertyAccessToken()
+    {
+        // TODO: implement
+        $this->markTestIncomplete('Not implemented');
```
Comment (tokuhirom) [inline src/clients/channel-access-token/test/Model/IssueChannelAccessTokenResponseTest.php:103]: nits: If there's no plan to implement here, you can skip the test code generation.

**Comment by tokuhirom** [comment]
- I suggest to bump the major version for next release
- I recommend to generate the code on github actions

**Comment by moririnson** [comment]
Follow https://github.com/line/line-openapi and automatically generate code in a separate pull-request!
https://github.com/line/line-bot-sdk-php/issues/461

File: src/clients/channel-access-token/test/Model/IssueChannelAccessTokenResponseTest.php (hunk 3 above)
Comment (moririnson) [reply src/clients/channel-access-token/test/Model/IssueChannelAccessTokenResponseTest.php]: Thanks. ignore tests using `.openapi-generator-ignore`.

**Review by satosby — APPROVED** [review]
LGTM

**Comment by moririnson** [comment]
thank you~

**Pull request merged**